doc/api/errors.md

```diff
@@ -450,18 +450,35 @@ added properties.
 
 #### error.code
 
-Returns a string representing the error code, which is always `E` followed by
+Is a string representing the error code, which is always `E` followed by
 a sequence of capital letters, and may be referenced in `man 2 intro`.
 
 #### error.errno
 
-Returns a number corresponding to the **negated** error code, which may be
+Is a number corresponding to the **negated** error code, which may be
 referenced in `man 2 intro`. For example, an `ENOENT` error has an `errno` of
 `-2` because the error code for `ENOENT` is `2`.
 
+In some modules, this property is the same string as `error.code`.
```

Comment:
I'm not familiar enough with the libuv error codes, but I would guess that the `.code` would be the same across platforms whereas the `.errno` may not be?